### PR TITLE
[JENKINS-59291] force resources plugin to 2.7

### PR DESCRIPTION
**Pull request by olamy**
Signed-off-by: olivier lamy <olamy@apache.org>

# Description

See [JENKINS-59291](https://issues.jenkins-ci.org/browse/JENKINS-59291).

# Submitter checklist
- [x] Link to JIRA ticket in description, if appropriate.
- [x] Change is code complete and matches issue description
- [x] Appropriate unit or acceptance tests or explanation to why this change has no tests
- [ ] Reviewer's manual test instructions provided in PR description. See Reviewer's first task below.

# Reviewer checklist
- [x] Run the changes and verified the change matches the issue description
- [ ] Reviewed the code
- [ ] Verified that the appropriate tests have been written or valid explanation given

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -106,6 +106,16 @@
...
       <plugins>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-resources-plugin</artifactId>
+          <!--
+          maven totally change his way to copy symlink
+          this mean the symlink is copied as a symlink and not anymore as a new file
+          see JENKINS-59291
+          -->
+          <version>2.7</version>
```
Comment (bitwiseman) [inline pom.xml]: So, this is to keep us from using `3.1.0` or is it meant to make us use at least `2.7`?

Comment (bitwiseman) [reply pom.xml]: Ah: 
https://stackoverflow.com/questions/40346225/maven-resources-plugin-symbolic-link-handling
https://issues.apache.org/jira/browse/MRESOURCES-237

Comment (bitwiseman) [reply pom.xml]: Is there anyway to test this change within this plugin?  Can we add a test the checks the symlink behavior so it doesn't break in the future?

**Comment by olamy** [comment]
still a draft PR the idea is to add a test on the jar content to ensure the js file is here.
Ideally I will fix https://issues.apache.org/jira/browse/MRESOURCES-237

**Comment by olamy** [comment]
@bitwiseman test added

**Pull request merged**